src/lwc/geBatchGiftEntryHeader/__tests__/geBatchGiftEntryHeader.test.js

```diff
@@ -64,35 +52,73 @@ describe('c-ge-batch-gift-entry-header', () => {
 
     describe('gift entry batch header elements', () => {
         it('should render three action buttons', async () => {
-            getGiftBatchTotalsBy.mockResolvedValue(APEX_GIFT_BATCH_TOTALS_BY_SUCCESS);
             const element = setup();
 
+            element.batchTotals = {
+                hasValuesGreaterThanZero: true,
+                hasPaymentsWithExpiredAuthorizations: false,
+                totalGiftsCount: 20,
+                processedGiftsCount: 10,
+                failedPaymentsCount: 0,
+                failedGiftsCount: 5,
+                expiredPaymentsCount: 0
+            }
             await flushPromises();
             const buttons = element.shadowRoot.querySelectorAll('lightning-button');
             expect(buttons.length).toBe(3);
         });
 
         it('renders detail row', async () => {
-            getGiftBatchTotalsBy.mockResolvedValue(APEX_GIFT_BATCH_TOTALS_BY_SUCCESS);
             const element = setup();
 
+            element.batchTotals = {
+                hasValuesGreaterThanZero: true,
+                hasPaymentsWithExpiredAuthorizations: false,
+                totalGiftsCount: 20,
+                processedGiftsCount: 10,
+                failedPaymentsCount: 0,
+                failedGiftsCount: 5,
+                expiredPaymentsCount: 0
+            }
+
+            isElevateCustomerAdapter.emit(false);
+
             await flushPromises();
             const headerDetailRows = element.shadowRoot.querySelectorAll('c-util-page-header-detail-row');
             expect(headerDetailRows.length).toBe(1);
         });
 
         it('does not render detail row', async () => {
-            getGiftBatchTotalsBy.mockResolvedValue(APEX_GIFT_BATCH_TOTALS_WITHOUT_ROWS);
             const element = setup();
 
+            element.batchTotals = {
+                hasValuesGreaterThanZero: false,
+                hasPaymentsWithExpiredAuthorizations: false,
+                totalGiftsCount: 20,
+                processedGiftsCount: 0,
+                failedPaymentsCount: 0,
+                failedGiftsCount: 0,
+                expiredPaymentsCount: 0
+            }
+
             await flushPromises();
             const headerDetailRows = element.shadowRoot.querySelectorAll('c-util-page-header-detail-row');
             expect(headerDetailRows.length).toBe(0);
         });
 
         it('renders detail blocks for records processed and records failed with correct record counts on load', async () => {
```

Comment:
I think we could duplicate this jest test to add coverage for the Elevate payment-related page header detail blocks.

Reply:
@psadie-sfdo - I might be mis-reading this, but I believe the elevate-specific header details aren't yet visible in this WI? That story is still in the backlog I believe!

Reply:
🤦‍♂️ sorry, that's right! Resolved!